ImportExtension/AddImportOperation.swift

```diff
@@ -120,9 +120,12 @@ class AddImportOperation {
         self.buffer.lines.insert(importString, at: line)
         
         //add a new selection. Bug fix for #7
-        let selectionPosition = XCSourceTextRange.init(start: XCSourceTextPosition.init(line: 0, column: 0), end: XCSourceTextPosition.init(line: 0, column: 0))
```

Comment:
I'm wondering now if reverting the code pushed to fix #7 would just work for Xcode 9 instead of adding anything new, because I remember it worked fine before that, bar the crash. What worries me as well Xcode 8.x will still be around for at least 2 more months, should we add a regression bug in now?